app/dev_mode/views.py

```diff
@@ -38,11 +38,26 @@ def dev_mode():
         return_by = form.get("return_by")
         employment_date = form.get("employment_date")
         region_code = form.get("region_code")
+        language_code = form.get("language_code")
         sexual_identity = form.get("sexual_identity")
         variant_flags = {"sexual_identity": sexual_identity}
-        payload = create_payload(user, exp_time, eq_id, period_str, period_id, form_type, collection_exercise_sid,
-                                 ref_p_start_date, ref_p_end_date, ru_ref, ru_name, trad_as, return_by, employment_date,
-                                 region_code, variant_flags)
+        payload = create_payload(user=user,
```

Comment:
This looks a lot clearer as explicitly named parameters, maybe this is all it needed!